src/components/Film.js

```diff
@@ -10,25 +78,24 @@ const Film = (film) => {
       return RATE_CLASS + 'red';
     }
   };
-
   return (
     <div className="movie">
-      <div className="movie__cover-inner">
-        <img
-          src={film.film.posterUrlPreview}
-          className="movie__cover"
-          alt={film.film.nameRu}
-        />
-        <div className="movie__cover--darkened"></div>
-      </div>
-      <div className="movie__info">
-        <div className="movie__title">{film.film.nameRu}</div>
-        <div className="movie__category">
-          {film.film.genres.map((genre, key) => `  ${genre.genre},`)}
+      <Link to="/moviesApp-React/aboutFilm" onClick={event}>
```

Comment:
what happens in this case?
![image](https://user-images.githubusercontent.com/3474059/143675125-af4643ae-3ed8-402c-a08c-47b768bd1fb0.png)